### PR TITLE
OG-907: Remove calls to 'getNetworkType' in ContractInteractor

### PR DESCRIPTION
**Pull request by forshtat**

### CODE REVIEW

File: packages/dev/test/RelayServerRequestsProfiling.test.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { createServerLogger } from '@opengsn/logger/dist/ServerWinstonLogger'
 
 contract('RelayServerRequestsProfiling', function (accounts) {
   const refreshStateTimeoutBlocks = 2
-  const callsPerStateRefresh = 17
```
Comment (drortirosh) [inline packages/dev/test/RelayServerRequestsProfiling.test.ts:11]: please explain how comes there is one more RPC call after REMOVING a call to RPC api.

File: packages/dev/test/RelayServer.test.ts [hunk 2]
```diff
@@ -288,8 +288,7 @@ contract('RelayServer', function (accounts: Truffle.Accounts) {
           assert.fail()
         } catch (e: any) {
           assert.include(e.message,
-            // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-            `Network ${env.relayServer.contractInteractor.getNetworkType()} doesn't support eip1559`)
+            'Current network does not support EIP-1559 transactions.')
```
Comment (drortirosh) [inline packages/dev/test/RelayServer.test.ts:291]: chainId?

Comment (forshtat) [reply packages/dev/test/RelayServer.test.ts]: Fixed.

**Review by drortirosh — APPROVED** [review]

File: packages/common/src/ContractInteractor.ts [hunk 3]
```diff
@@ -230,6 +229,13 @@ export class ContractInteractor {
     if (block.baseFeePerGas != null) {
       this.logger.info('Network supports Type 2 Transactions (EIP-1559). Checking RPC node \'eth_feeHistory\' method')
       try {
+        // hard-coded default values as these are not used but compared to 'gasPrice' and logged
+        const { baseFeePerGas, priorityFeePerGas } = await this.getGasFees(5, 50)
```
Comment (drortirosh) [inline packages/common/src/ContractInteractor.ts:233]: just read the description of the PR, which is the only place that currently give meaning to these "5" and "50".... please make them constants.

File: packages/common/src/ContractInteractor.ts [hunk 4]
```diff
@@ -230,6 +229,12 @@ export class ContractInteractor {
     if (block.baseFeePerGas != null) {
       this.logger.info('Network supports Type 2 Transactions (EIP-1559). Checking RPC node \'eth_feeHistory\' method')
       try {
+        const { baseFeePerGas, priorityFeePerGas } = await this.getGasFees(constants.getGasFeesBlocks, constants.getGasFeesPercentile)
```
Comment (drortirosh) [inline packages/common/src/ContractInteractor.ts:232]: if you're using here the constants, then what's the use of having them configured at all?

**Pull request merged**